### ISSUE TITLE
Symfony: `environment: 'prod'` not working

### ISSUE THREAD

**Issue by ThomasLandauer** [Codeception/module-symfony#88]
#### What are you trying to achieve?

Run tests in Symfony's PROD environment.

#### What do you get instead?

Tests are still run in TEST environment.

### Details

To investigate this, I created a controller which just outputs the environment (`return new Response($kernel->getEnvironment());`). When browsing to its url I get whatever I have in Symfony's `.env`. Fine.

When I access the url with codeception (`codecept_debug($I->grabPageSource());`), I get whatever I write into `functional.suite.yml`, even an arbitrary string like `foo`.

However, if I put `prod`, the app isn't really run in PROD environment. The feature I want to test is if the correct error page is shown. But in DEV/TEST environment Symfony always shows its internal Exception Page, not my own error page (see https://symfony.com/doc/current/controller/error_pages.html#testing-error-pages-during-development ) - that's how I know that I am still in TEST environment in fact.

I even tried with a `@prepare` annotation (`$symfony->_reconfigure(['environment' => 'prod']);`) to switch just for this single test (cause that's what I originally intended).

* Codeception version: 2.5.2
* Symfony version: 4.2.1
* Suite configuration:

```yml
actor: FunctionalTester
modules:
    enabled:
        - Symfony:
            app_path: 'src'
            environment: 'prod'
```

**Comment by TavoNiievez** [comment]
@ThomasLandauer Can you reproduce this problem in the test project?

I need to know if this is still a problem with the supported versions of Symfony, at least Codeception 4 and version 1.6 of this module.

I am not able to reproduce or fix the problem with the versions described in this issue, so please update it.

**Comment by ThomasLandauer** [comment]
Yes, the issue is still there. I created https://github.com/ThomasLandauer/symfony-module-tests/tree/issue_88
Run `vendor/bin/codecept run Functional EnvironmentCest:try -vvv`, and you'll see "foo" in the output, which is the string I put in `Functional.suite.yml`.

**Comment by TavoNiievez** [comment]
This issue will be fixed with version 3.4.0 of the module.
A new config parameter will be added:


```yml
# tests/Functional.suite.yml
modules:
    enabled:
        - Symfony:
              bootstrap: true
```

Then you have to make sure that your `codeception.yml` file is not loading `.env` files.

You will have to have an `.env.prod` file and enable it in the` Funciontal.suite.yml` file as well.
You will have to enable the services that are not public for the `prod` environment in `config/services_prod.php`, you will also have to enable the necessary bundles for the symfony module to run the tests well (_WebProfiler_ for example) in your `config/bundles.php` file.

After doing all this you will have to move the necessary settings in your `config/packages/prod/` and `config/routes/prod/` configuration files.

I'm still not quite clear on the use case where you would want to run tests in this environment, but at least it's already possible to do so.
